MultiLabel/Core/src/main/java/org/tribuo/multilabel/MultiLabel.java

```diff
@@ -277,6 +279,43 @@ public String getSerializableForm(boolean includeConfidence) {
         return str;
     }
 
+    /**
+     * Converts this MultiLabel into a DenseVector using the indices from the output info.
+     * The label score is used as the value for that index if it's non-NaN, and is 1.0 otherwise.
+     * Labels which are not present are given the score 0.0.
+     * @param info The info to use for the ids.
+     * @return A DenseVector representing this MultiLabel.
+     */
+    public DenseVector convertToDenseVector(ImmutableOutputInfo<MultiLabel> info) {
+        if (!(info instanceof ImmutableMultiLabelInfo)) {
+            throw new IllegalStateException("Unexpected info type, found " + info.getClass().getName() + ", expected " + ImmutableMultiLabelInfo.class.getName());
+        } else {
+            ImmutableMultiLabelInfo imInfo = (ImmutableMultiLabelInfo) info;
+            Set<Integer> seenIndices = new HashSet<>(labels.size());
+            double[] values = new double[imInfo.size()];
+
+            for (Label l : labels) {
+                int i = imInfo.getID(l.getLabel());
+                if (i != -1) {
+                    if (!seenIndices.contains(i)) {
+                        double score = l.getScore();
+                        if (Double.isNaN(score)) {
```

Comment:
This is fairly confusing and counterintutive. It should at least be documented in the code, or preferably in the JavaDocs.

Reply:
I fixed the javadoc on `MultiLabel` to reflect how ground truth values are stored.